README.md

```diff
@@ -1,47 +1,42 @@
 # **Portshift Orb for CircleCi**
 
-## **What is Portshift Orb?**
-With this orb you can easily integrate Portshift identity creation and vulnerabilities scans into your CircleCI Workflows.
-Portshift identity creation is essential part of Portshift runtime security, authenticating and authorizing containers build and deployments with vulnerabilities scan results. The scan engine is currently based on [Clair](https://github.com/quay/clair), in the future additional scan engines will be added to Portshift Orb to enhance the vulnerabilities information. 
-Portshift Orb is an easy way to get started with Portshift runtime security platform, the image identity and the vulnerabilities scan results will be shown in Portshift Management platform.
+## **What is the Portshift Orb?**
```

Comment:
David, we want to show higher value than vulns. scanning. 
That's why we added identity creation (in the future we'll add also meta-data from CD stage).
So, it's kept broad in purpose.